galvan-support/src/main/java/org/terracotta/testing/client/GalvanClientTestEnviroment.java

```diff
@@ -0,0 +1,71 @@
+package org.terracotta.testing.client;
+
+import org.terracotta.passthrough.IClientTestEnvironment;
+import org.terracotta.passthrough.IClusterInfo;
+import org.terracotta.passthrough.IServerInfo;
+import org.terracotta.testing.master.ClusterInfo;
+import org.terracotta.testing.master.ServerInfo;
+import org.terracotta.testing.rules.Cluster;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * @author vmad
+ */
+public class GalvanClientTestEnviroment implements IClientTestEnvironment {
+
+  private final String clusterUri;
+  private final ClusterInfo clusterInfo;
+  private final int totalClientCount;
+  private final int thisClientIndex;
+
+  public GalvanClientTestEnviroment(String clusterUri, ClusterInfo clusterInfo, int totalClientCount, int thisClientIndex) {
+    this.clusterUri = clusterUri;
+    this.clusterInfo = clusterInfo;
+    this.thisClientIndex = thisClientIndex;
+    this.totalClientCount = totalClientCount;
+  }
+
+  @Override
+  public String getClusterUri() {
+    return this.clusterUri;
+  }
+
+  @Override
+  public int getTotalClientCount() {
+    return this.totalClientCount;
+  }
+
+  @Override
+  public int getThisClientIndex() {
+    return this.thisClientIndex;
+  }
+
+  @Override
+  public IClusterInfo getClusterInfo() {
+    return new IClusterInfo() {
```

Comment:
If the implementation of IClusterInfo is moved out, then you could just use the existing implementation of IClientTestEnvironment from the passthrough project.